BD-1281: Clarifying Roku IAM is not OOTB (#3291)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/roku/in-app_messaging/integration.md

# In-app messaging implementation guide

> This implementation guide covers in-app message code considerations and accompanying code snippets. While we provide a sample out-of-the-box implementation, you will need to build a container for your payload and most likely implement custom code for your desired UI. Because your code will be unique to your app, you do not need to handle all situations listed below if not relevant to your use case. For example, if you don't use delayed display of in-app messages, you will not need to implement that logic and edge cases.

Reviewer: ```suggestion
This implementation guide covers in-app message code considerations and accompanying code snippets. While we provide sample integration code, you will need to add logic to handle and display triggered messages within your desired UI. 

Because your code will be unique to your app, you do not need to handle all situations listed below if not relevant to your use case. For example, if you don't use delayed display of in-app messages, you will not need to implement that logic and edge cases.
```
